core/src/main/java/org/bitcoinj/wallet/Protos.java

```diff
@@ -16527,6 +16531,10 @@ private Wallet() {
        */
       public static final int ENCRYPTED_ECDH_KEYEXCHANGE_AES_VALUE = 4;
       /**
+       * <pre>
+       * Not used for wallet
+       * </pre>
+       *
```

Comment:
Protos.java is a generated file from wallet.proto.  One could ask why we include it in the repo.  In bitcoinj it was removed from the repo (since it was autogenerated), but then those developers decided to add it back.